### PR TITLE
Add blogpost

### PR DESCRIPTION
**Pull request by errorxyz**

### CODE REVIEW

File: src/content/posts/releases/mitmproxy-11/index.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+Our transparent proxy modes now all support HTTP/3 as well:
+
+```shell
+$ mitmproxy --mode wireguard
+$ mitmproxy --mode local
+$ mitmproxy --mode transparent
+```
+
+We have successfully tested HTTP/3 support with Firefox, Chrome, various cURL builds, and other clients to iron out 
+compatibility issues.
```
Comment (errorxyz) [inline src/content/posts/releases/mitmproxy-11/index.md:44]: just curious, what other clients have we tested on?

File: src/content/posts/releases/mitmproxy-11/index.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+The only limitation we are aware of is that Chrome [does not trust user-added Certificate Authorities for QUIC].
+This means your will either need to provide a publicly trusted certificate (e.g. from Let's Encrypt), start Chrome with
```
Comment (errorxyz) [inline src/content/posts/releases/mitmproxy-11/index.md:46]: ~your~ you

File: src/content/posts/releases/mitmproxy-11/index.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+[Encrypted Client Hello]: https://blog.cloudflare.com/announcing-encrypted-client-hello/
+
+```shell
+$ mitmproxy --mode local --set experimental_transparent_http3=true
+```
+
+Any cURL, Chromium or Firefox based HTTP/3 client ([setup]) can be used to access these features. There are still some
+bugs that are yet to be fixed which you can find in our [bug-tracker].
+
+[setup]: https://github.com/mitmproxy/mitmproxy/issues/7025#issuecomment-2351138170
```
Comment (errorxyz) [inline src/content/posts/releases/mitmproxy-11/index.md:80]: I think we should mention this comment somewhere - it'd save a lot of debugging time :)

**Review by errorxyz — COMMENTED** [review]
Thanks for your review! Blog definitely looks better now🤩

File: src/content/posts/releases/mitmproxy-11/index.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+*Hi! I'm [Gaurav Jain], one of the students selected for this year's Google Summer of Code program to work on mitmproxy.
+During this summer, I've worked on improving various low-level networking parts of mitmproxy some of which include
+HTTP/3 and DNS. You can find my project report [here].*
+
+[Gaurav Jain]:  {{< relref "/authors/gaurav-jain" >}}
+[here]: https://gist.github.com/errorxyz/af6f26549e9122f3ff3b93fd9d257df1
+
+## HTTP/3
+
+HTTP/3 now "just works" for reverse proxies. Your mitmproxy instance will listen for
```
[the remark this answers is not in the document]
Comment (mhils) [reply src/content/posts/releases/mitmproxy-11/index.md:28]: No additional `http3://` extra mode shenanigans required or anything. If the destination advertises h3 support, clients can upgrade.

File: src/content/posts/releases/mitmproxy-11/index.md (hunk 1 above)
Comment (mhils) [reply src/content/posts/releases/mitmproxy-11/index.md:44]: aioquic, reqwest :)

**Comment by mhils** [comment]
Final polish, please take a look! 😃

**Review by errorxyz — COMMENTED** [review]
LGTM🎉

**Pull request merged**